src/pages/upload/index.tsx

```diff
@@ -102,6 +129,25 @@ const Upload: React.FC = () => {
           </div>
         )
       })}
+      {ability.can('upload', 'Video.WithInstructor') ? (
```

Comment:
In addition to publishers/editors, eventually _instructors_ will be able to use this Course Builder. Those with only the `instructor` role won't be able to select the instructor, hence the ability check.